wire-library/wire-runtime-swift/src/test/proto/type.proto

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+syntax = "proto2";
+
+enum Type {
+  UNKNOWN = 0;
+  VALUE = 1;
+}
```

Comment:
What's that for?

Reply:
Prior to bumping swiftpoet to the latest version, the `:wire-runtime-swift:generateSwiftProtos` gradle task would  fail when attempting to generate the swift version of that proto. I left it in as a "pseudo" integration test. Happy to remove it if you think it's uanessacry or add a unit test specifically tested the generated swift file from the proto.

Reply:
Then please add a comment to the proto file on the type explaining why it's there. And it's good!

Reply:
Done!